src/NServiceBus.MessagingBridge/Heartbeats/HeartbeatSender.cs

```diff
@@ -8,6 +8,7 @@
 using ServiceControl.Plugin.Heartbeat.Messages;
 using Transport;
 
+[Janitor.SkipWeaving]
```

Comment:
This was needed after introduction of Janitor.Fody - it complained about the dispose method having a body